### PR TITLE
Updating requirements/environment

### LINKED ISSUE #141: Update requirements.txt
Update requirements.txt so we can pip install what we need

### PR DESCRIPTION
**Pull request by Will-Cooper**
closes #141, simplify requirements.txt and environment.yml, swap mult…multiprocessing for multiprocess to work on Mac.
Can test with:
```bash
conda env create --name simpletest -f environment.yml
conda activate simpletest
python -m simple_app.app_simple
conda deactivate
conda env remove -n simpletest
```

### CODE REVIEW

**Comment by dr-rodriguez** [comment]
I like the simplification, but we should considering pinning the requirement versions, especially since this is the website. The general user can grab the latest, but the website should have pinned versions or an update can break it.

**Comment by Will-Cooper** [comment]
I agree, I'm also considering upgrading the python version on the website to get ahead of the game. Just need to do some fixes as there's several changes in pandas etc.

File: environment.yml [hunk 1]
```diff
@@ -1,138 +1,11 @@
 name: simple
 channels:
-  - http://ssb.stsci.edu/astroconda
   - defaults
+  - conda-forge
+  - http://ssb.stsci.edu/astroconda
```
Comment (dr-rodriguez) [inline environment.yml:5]: Do we actually need conda-forge and astroconda? I thought the later was actually deprecated.

File: requirements.txt [hunk 2]
```diff
@@ -1,97 +1,18 @@
...
-zipp>=3.4.1
+astrodbkit2>=0.5.1
+astropy>=5.2.0
+bokeh==3.0.2
+flask>=3.0.1
+flask-cors>=4.0.0
+flask-wtf>=1.2.1
+markdown2>=2.4.12
+multiprocess>=0.70.16
+numpy>=1.24.0
```
Comment (dr-rodriguez) [inline requirements.txt:9]: We should provide an upper bound for the numpy version. Numpy 2.0 is coming soon and could break things.

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**